src/main/java/Chandler.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Chandler {
+
+    public static final String LINE_DIVIDER = "____________________________________________________________";
+    public static final String OUTPUT_INDENTATION = "    ";
+
+    public static void main(String[] args) {
```

Comment:
Avoid long methods. Take corrective action when the method exceeds 30 Lines of Code.